[quant][eagermode] Add additional_fuser_method_mapping to config

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#46355 [quant][eagermode] Add additional_fuser_method_mapping to config**

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

jerryzh168 committed:
[quant][eagermode] Add additional_fuser_method_mapping to config

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

vkuzo reviewed (approved):
lg, should we add a test?

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][eagermode] Add additional_fuser_method_mapping to config"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24319562](https://our.internmc.facebook.com/intern/diff/D24319562)

[ghstack-poisoned]
